### PR TITLE
Handling code errors used in the frontend

### PR DESCRIPTION
**Pull request by anistal**
#### Description
- Error codes that will be used in the frontend to handle errors.
- Deleted icon property from FragmentModel.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@
         <module>plugins</module>
         <module>serving-api</module>
         <module>doc</module>
-        <module>web</module>
+	<!-- <module>web</module> -->
```
Comment (danielcsant) [inline pom.xml]: uncomment this line

File: serving-api/pom.xml [hunk 2]
```diff
@@ -35,10 +35,12 @@
     </properties>
 
     <dependencies>
+	    <!--
         <dependency>
             <groupId>com.stratio.sparkta</groupId>
             <artifactId>web</artifactId>
         </dependency>
+	    -->
         <dependency>
```
Comment (danielcsant) [inline serving-api/pom.xml]: uncomment this

File: pom.xml (hunk 1 above)
Comment (anistal) [inline pom.xml]: The project do not compile in my local if these lines are not commented. Was not it fixed?

File: examples/policies/ITwitter-OPrintWithFragments.json [hunk 3]
```diff
@@ -1,5 +1,7 @@
 {
   "name": "policy-Twitter-Print",
+  "description": "policy-Twitter-Print",
+  "description": "Input twitter output print",
```
Comment (antonioalf) [inline examples/policies/ITwitter-OPrintWithFragments.json]: duplicated description line

**Pull request merged**